SWLOR.Game.Server/Feature/LootTableDefinition/DantooineLootTableDefinition.cs

```diff
@@ -0,0 +1,507 @@
+        {
+            _builder.Create("DANTOOINE_DANTARI_SHAMAN")
+                .AddItem("fiberp_imperfect", 5)
+                .AddItem("fiberp_high", 10)
+                .AddItem("lth_imperfect", 5)
+                .AddItem("lth_high", 10)
+                .AddItem("bread_flour", 5);
+
+            _builder.Create("DANTOOINE_DANTARI_SHAMANGEAR")
+                .AddItem("kwi_knife", 20)
+                .AddItem("kwi_greatsword", 20)
+                .AddItem("kwi_longsword", 20)
+                .AddItem("kwi_electroblade", 20)
+                .AddItem("kwi_katar", 20)
+                .AddItem("kwi_staff", 20)
+                .AddItem("kwi_twinblade", 20)
+                .AddItem("kwi_twinelec", 20)
+                .AddItem("kwi_spear", 20)
+                .AddItem("kwi_pistol", 20)
+                .AddItem("kwi_rifle", 20)
+                .AddItem("kwi_shield", 10)
+                .AddItem("kwi_h_cloak", 10)
+                .AddItem("dhcl005k", 10)
+                .AddItem("dlcl005k", 10)
+                .AddItem("kwi_hyper_cloak", 10)
+                .AddItem("kwi_light_cloak", 10)
+                .AddItem("dhbe005k", 10)
+                .AddItem("dlbe005k", 10)
+                .AddItem("kwi_heavy_belt", 10)
+                .AddItem("kwi_hyper_belt", 10)
+                .AddItem("kwi_light_belt", 10)
+                .AddItem("dhrg005k", 10)
+                .AddItem("dlrg005k", 10)
+                .AddItem("kwi_heavy_ring", 10)
+                .AddItem("kwi_hyper_ring", 10)
+                .AddItem("kwi_light_ring", 10)
+                .AddItem("dhnk005k", 10)
+                .AddItem("dlnk005k", 10)
+                .AddItem("kwi_heavy_neck", 10)
+                .AddItem("kwi_hyper_neck", 10)
+                .AddItem("kwi_light_neck", 10)
+                .AddItem("dhbr005k", 10)
+                .AddItem("kwi_heavy_bracer", 10)
+                .AddItem("dlbr005k", 10)
+                .AddItem("kwi_hyper_gloves", 10)
+                .AddItem("kwi_light_gloves", 10)
+                .AddItem("dhlg005k", 10)
+                .AddItem("dllg005k", 10)
+                .AddItem("kwi_heavy_leg", 10)
+                .AddItem("kwi_hyper_boots", 10)
+                .AddItem("kwi_light_boots", 10);
+
+            _builder.Create("DANTOOINE_DANTARI_SHAMAN_RARES")
+                .AddItem("kwi_heavy_armor", 1)
+                .AddItem("kwi_hyper_tunic", 1)
+                .AddItem("kwi_light_tunic", 1)
+                .AddItem("dhar005k", 10)
+                .AddItem("dlar005k", 10)
+                .AddItem("dhhl005k", 10)
+                .AddItem("dlhl005k", 10)
+                .AddItem("kwi_heavy_helm", 10)
+                .AddItem("kwi_hyper_cap", 10)
+                .AddItem("kwi_light_cap", 10);
+
+            _builder.Create("DANTOOINE_DANTARI_SHAMAN_RARES")
+                .IsRare()
+                .AddItem("ruby", 99, 1, true)
+                .AddItem("map_70", 2, 1, true)
```

Comment:
I think every single loot table you have has these exact same maps set up. I would recommend changing it up a bit so that a few different ones are on each. This incentivizes players to hunt down different enemies.